### ISSUE TITLE
ip address

### ISSUE THREAD

**Issue by vejan738** [chickey/RS485-WiFi-EPEver#26]
the IP address that shows up 192.168 4.*** is wrong for my network, I cant connect to it. How to change network IP for my network 192.168.1.****?

**Comment by Sundancer78** [comment]
Unfortunately you don't have many options. The WIFI adapter (I assume you have the 2.4G RJ45 D version?) apparently only works with the Solar Guardian app (I have the Android version. With this app, the WiFi adapter can be integrated into your home WLAN. You add Add a gateway, so you have to scan the QR code that is on the back of the WiFi adapter. Then the charging controller is selected, depending on what you have, and then you have (or before selecting the charging controller? I know it not exactly.) the option to select your home WLAN. This can also be done later in the Solar Guardian app.
With another app, for example EPEVER Pair (local), you can only connect to the Access Point of the Wifi adapter. Unfortunately, I never got any further. Actually, the charging controller should be identified automatically, but that doesn't work. Selecting it manually doesn't work properly either, as no data can be retrieved at the end. For me, the 2.4G RJ45 D adapter was never recognized as such. Once it was a 2.4G PCB D? Adapter, sometimes an RJ45 A or whatever. At some point I noticed that the EPEVER PAir app contains the RJ45 D version, but only the RJ45 C version, which I couldn't even really find on the Internet, as if this adapter only existed on paper.
Other apps from EPEVER, such as the apps for various charging controllers, also seem to require a different version of the WiFi adapter in order to establish a connection successfully. I'm guessing the Wifi 2.4G eBox. I can't say exactly because I don't have these devices. What I can say is that I didn't get the BT RJ45 D version to work either (ChargeController (Sealed) or (Li)) app. Apparently the Wifi eBox or the eBox-BLE-01 Converter is required here. You still have the option of using the RJ45 - USB adapter cable to at least be able to access the PC software.
In my opinion, what EPEVER has fabricated here is terrible.
I have now solved the access to my Tracer AN in this way: I connected the RS485-1M2S distribution box to the Tracer and the WIFI 2.4G RJ45 D adapter at one output in order to have a permanent connection to the Solard Guardian app or to the cloud. At the other end I connected the EPEVER TCP 306 box so that I could access the “Solar Station Monitor” software on the PC via LAN. I was just tired of constantly replugging the RJ45 USB cable with the WIFI adapter because the Solar Station Monitor software doesn't upload the data to the cloud, at least not to the Solar Guardian Cloud. The PC software Solar Guardian also apparently does not load the data into the cloud. Otherwise, this software only seems to connect via an RJ45 USB adapter cable. At least I cannot establish a connection with the Solar Guardian software via the virtual COM interface that I need via the TCP 306 converter for Solar Station Monitor. I was completely fed up and didn't try any further after I was disappointed to find that the TCP 306 converter cannot operate the Solar Guardian Cloud and the Solar Station Monitor software for the PC at the same time. Hence the combination of RS485-1M2S distribution box, TCP 306 converter and WIFI 2.4G RJ45 D adapter.

So now I have written to you with my experiences. I hope I was able to help you with your problem too.

**Comment by chickey** [comment]
Hi sundancer, i'm not sure which adaptors you are referring too but my hardware is designed specifically for my custom software or esphome and is not designed to work with any other software.

**Comment by Sundancer78** [comment]
> Hi sundancer, i'm not sure which adaptors you are referring too but my hardware is designed specifically for my custom software or esphome and is not designed to work with any other software.

oh shit!
I have to admit, I didn't look at the Github. I got an email (probably I had activated your Git notifications) with the problem from @vejan738 and happily wrote down my answer. I use the EPEVER adapters! :-D
But hey! If one or the other of my odyssey reads through the adapter world of EPEVER, they will try out your variant. I probably should have done that too. Sorry I didn't turn my head on first to check what this is actually about.
